components/automate-ui/src/app/components/create-object-modal/create-object-modal.component.scss

```diff
@@ -27,6 +27,10 @@ chef-modal {
 
       .checkbox-margin {
         margin-bottom: 30px;
+
+        #add-policy-checkbox {
+          margin-bottom: 30px;
+        }
```

Comment:
I don't get it. What does the class thing do? Is this added (to 60px), effectively?

Reply:
Checkbox margin contains two checkboxes. This is adding a margin below the first checkbox whereas the checkbox margin is adding a margin below the container relative to the rest of the modal.